packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/.eslintrc

```
@@ -0,0 +1,7 @@
+{
+	"extends": "@pantheon-systems/eslint-config-decoupled-kit",
```

I would expect the react config here. Also you can omit the `eslint-config` prefix here if you'd like
```suggestion
	"extends": "@pantheon-systems/decoupled-kit/react",
```

---

This will probably cause a bunch of 'prop-types' errors. I believe this is because Gatsby ignores this rule or handles it somehow under the hood. I think for now it's worth ignoring this rule. When we figure out our TypeScript story for the starters, we can revisit this.